package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "build": "webpack",
-    "dev": "webpack serve --open"
+    "dev": "webpack serve --open",
+    "deploy": "gh-pages -d dist"
```

Comment:
- [ ] Please, consider that you `package.json` file -> `scripts` section is missing the `start` script. Because of this issue `npm start` is not working in your project and gives an error like this:
<img width="496" alt="Screen Shot 2022-08-24 at 00 27 24" src="https://user-images.githubusercontent.com/49064106/186259172-d84bf991-b7e8-4e14-b896-8782a9a1c843.png">

- To avoid this error, you have to add this script `"start": "webpack --config webpack.config.js --watch"` and then in the terminal the `npm install` command. After you can confidently use `npm start` without issues.

Reply:
Fixed, npm start can now run fine